cmd/celestia-appd/main.go

```diff
@@ -1,58 +1,15 @@
 package main
 
 import (
-	"io"
 	"os"
 
 	"github.com/celestiaorg/celestia-app/app"
-	"github.com/cosmos/cosmos-sdk/baseapp"
 	svrcmd "github.com/cosmos/cosmos-sdk/server/cmd"
-	servertypes "github.com/cosmos/cosmos-sdk/server/types"
-	"github.com/tendermint/spm/cosmoscmd"
-	"github.com/tendermint/tendermint/libs/log"
-	dbm "github.com/tendermint/tm-db"
 )
 
 func main() {
-	rootCmd, _ := cosmoscmd.NewRootCmd(
-		app.Name,
-		app.AccountAddressPrefix,
-		app.DefaultNodeHome,
-		app.Name,
-		app.ModuleBasics,
-		appBuilder,
-		// this line is used by starport scaffolding # root/arguments
-	)
-	if err := svrcmd.Execute(rootCmd, app.DefaultNodeHome); err != nil {
+	rootCmd := NewRootCmd()
+	if err := svrcmd.Execute(rootCmd, "CELESTIA", app.DefaultNodeHome); err != nil {
```

Comment:
Should the  "CELESTIA" string be in a const? (e.g. `envPrefix`)

Reply:
good idea, https://github.com/celestiaorg/celestia-app/pull/399/commits/740ad0045ce96817e5353b972ffed6f088aadc1f